prow/spyglass/lenses/lenses.go

```diff
@@ -91,9 +91,6 @@ func RegisterLens(lens Lens) error {
 	if config.Title == "" {
 		return errors.New("empty title field in view metadata")
 	}
-	if config.Priority < 0 {
```

Comment:
Why did you remove this?

Reply:
Priority is a uint so can't be less than zero.